p2p/chain.py

```diff
@@ -480,8 +482,24 @@ def request_receipts(self, target_td: int, headers: List[BlockHeader]) -> int:
             await self._handle_new_block(peer, cast(Dict[str, Any], msg))
         elif isinstance(cmd, eth.GetBlockHeaders):
             await self._handle_get_block_headers(peer, cast(Dict[str, Any], msg))
+        elif isinstance(cmd, eth.GetBlockBodies):
+            await handle_get_block_bodies(
+                self.db, peer, cast(List[Hash32], msg), self.logger, self.cancel_token)
+        elif isinstance(cmd, eth.GetReceipts):
+            await handle_get_receipts(
+                self.db, peer, cast(List[Hash32], msg), self.logger, self.cancel_token)
+        elif isinstance(cmd, eth.GetNodeData):
+            await handle_get_node_data(
+                self.db, peer, cast(List[Hash32], msg), self.logger, self.cancel_token)
+        elif isinstance(cmd, eth.Transactions):
+            # Transactions msgs are handled by our TxPool service.
+            pass
+        elif isinstance(cmd, eth.NodeData):
+            # When doing a chain sync we never send GetNodeData requests, so peers should not send
+            # us NodeData msgs.
+            self.logger.warn("Unexpected NodeData msg from %s", peer)
```

Comment:
Should we also disconnect from the peer?  Otherwise a malicious peer could flood the terminal with these messages.